controllers/Datarequest.php

```diff
@@ -67,17 +67,15 @@ public function view($requestId) {
         $tokenName = $this->security->get_csrf_token_name();
         $tokenHash = $this->security->get_csrf_hash();
 
-	# Get the data request and data request status from iRODS
-	$result = $this->api->call('uuGetDatarequest', ['requestId' => $requestId]);
-	$datarequest = json_decode($result->data->requestJSON);
+	# Get the data request status from iRODS
+	$result = $this->api->call('datarequest_get', ['request_id' => $requestId]);
```

Comment:
Is this needed before rendering the view? Now we call the same API function twice in a row (here and in the JS client code)

Reply:
As it is now, yes. The view/ view uses this view parameter to render the data request "progress bar" / status overview.

It should definitely be refactored though, because it's wasteful and ugly. That's a listed to-do, but it's outside the scope of the API rewrite IMO.